### PR TITLE
[JENKINS-41968] Fixed the pom for release management

### PR DESCRIPTION
**Pull request by Dohbedoh**
@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,8 +3,13 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.614</version>
+		<version>2.5</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Ideally it should be 2.30+

File: pom.xml [hunk 2]
```diff
@@ -3,8 +3,13 @@
...
 	</parent>
+
+	<properties>
+		<jenkins.version>1.614</jenkins.version>
```
Comment (oleg-nenashev) [inline pom.xml]: would be better to use LTS as a target baseline. E.g. 1.625.3

**Comment by Dohbedoh** [comment]
Hi @christ66 Can we have get a release of the plugin with that fix ?

**Comment by christ66** [comment]
@Dohbedoh can you create a new PR with the changes @oleg-nenashev has provided?

**Comment by Dohbedoh** [comment]
Sure. Created https://github.com/jenkinsci/semantic-versioning-plugin/pull/11

**Pull request merged**